### ISSUE TITLE
[Event] No Matter How Small (Rogue, pt. 1)

### ISSUE THREAD

**Issue by ben-lear** [AsdarDevelops/PokeRogue-Events#95]
### Event Name

No Matter How Small

### Rarity Tier

Rogue

### Waves

11-179

### Description

You encounter 3 Pokemon pups (Growlithe, Electrike, Buizel) in a cardboard box that says “Please help us find a home”.

There is a nearby town where someone might be able to care for them temporarily. You could also watch over them until a passerby can help. Or you can leave.

### Biomes

NON-EXTREME

### Options offered to the player

- [ ] Carry them to Town
    -  _You trek tirelessly to the town with the pups in tow, but it’s exhausting to carry them (they’re too tired to walk on their own)._
    - _Your pokemon assist you, and also become exhausted. -1 Def/-1 SpDef debuff for 5 floors.
    - _Unlocks following encounter._
    - _”You feel watchful eyes on you with the sense that something approves of your actions.”_
- [ ] Guard Them
    - _You stay and guard the pups for a while hoping to encounter another passerby._
    - _You are attacked by a <random biome monster>, but after a time someone comes to assist you!_
    - _Unlocks following encounter._
    - _”You feel watchful eyes on you with the sense that something approves of your actions.”_
- [ ] Abandon Them
    - _You’re an awful person but suffer no immediate “negative consequences”. Does not unlock following encounter_

### Explanation/Notes on Design

part 1 to the legendary beast trio quest chain 

### Notes to Artists

We need a sprite of Buizel/Electrike/Growlithe trio in a cardboard box.

**Comment by AsdarDevelops** [comment]
Some more feedback on this:

- Bring the waves to 11-79 instead, to avoid the questline starting too late.

- Add an additional option:

    - [ ] Haggle with the Grunt (optional but I think it's funny the player can start negotiation for the incarceration of these cuties)
        - 50/50 to earn even more money or trigger the fight.
        - _Does not unlock following encounter_ (even if it triggered the fight).

**Comment by ben-lear** [comment]
> Some more feedback on this:
> 
> * Bring the waves to 11-79 instead, to avoid the questline starting too late.
> * Add an additional option:
>   
>   * [ ]  Haggle with the Grunt (optional but I think it's funny the player can start negotiation for the incarceration of these cuties)
>     
>     * 50/50 to earn even more money or trigger the fight.
>     * _Does not unlock following encounter_ (even if it triggered the fight).

I think this may not be a good idea to significantly wave gate Rogue encounters, even if they are part of a chain. I understand gating to wave 150 maybe though.

**Comment by AsdarDevelops** [comment]
This should not be a Rogue though, this should be a Common, and we can play with the follow-up roll odds of the following parts:

- Part 1 - This (Common)
- Part 2 - (if Event continued) ("Great" but Special odds: say 50% in each CIVIL Encounter)-> The Reunion -> You meet the three PKMN again, they are fully evolved, and you decide to spend the most time with (__you choose one of them__, which marks which Beast you encounter)
- Part 3 - (if you continued the chain) ("Rogue" but Special odds: 30% in a Biome that makes sense for that particular Beast (I'd imagine Volcano, Lake and Power Plant or whatever)).

Part 3 is essentially Rogue due to the extra steps, but the kickstart Event shouldn't be. Otherwise this would be competing with Dark Deal and feel super bad to roll into imo. Since it's also 3 parts __and Common__, that's why I thought to cap it much lower.

TLDR: if it's part of a chain, it doesn't __have to be__ Rogue, even if it eventually leads to a legendary

**Comment by AsdarDevelops** [comment]
Big edit, should be finished now

**Comment by AsdarDevelops** [comment]
Hit this a little with the ban hammer. Candy option was way too good as it was __to every party member__
